### DISCUSSION TITLE
BTT EDDY Simple AF

### DISCUSSION THREAD

**Discussion by Newcomer81** [Guilouz/Creality-Helper-Script-Wiki#712] [General]
Hello everyone, 
I have a K1 Max and would like to get the BTT EDDY to run as Simple AF instead of the LIDAR. Now I have to install another custom Klipper.  But then the helper script macros will be lost. Is there another possibility or solution for this? Or how can I keep the macros working?
Is an implementation in helper script in the progress?

Thanks to all

**Comment by Schulli73** [comment]
I would like it if the BTT Eddy would be included in the helper script

**Comment by jglerner** [comment]
Head to this link:

 [https://www.reddit.com/r/crealityk1/comments/1di7nl6/btt_eddy_k1_guide/](url)

and to the README of this:

[https://github.com/ballaswag/creality_k1_klipper_mod](url)

While it is not integrated to Helper Script it shows you how to do what you want.
And, yes, it works for every member of  the Creality K1x's family
